utilities/unittests/test_oadp.py

```
@@ -18,76 +23,131 @@
 utilities.infra = mock_infra
 
 # Import after setting up mocks to avoid circular dependency
+from kubernetes.dynamic.exceptions import ResourceNotFoundError
+from ocp_resources.exceptions import ResourceTeardownError
+
+from utilities.constants import (
+    ADP_NAMESPACE,
+    LS_COMMAND,
+    TIMEOUT_20SEC,
+)
 from utilities.oadp import (  # noqa: E402
```

Ruff (0.14.11)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)